### ISSUE TITLE
No supported DataSource type found

### ISSUE THREAD

**Issue by smsajid** [redis/riot#140]
I am running in k8s:
`kubectl run -it --rm --restart=Never redis-riot --image=fieldengineering/riot --command -- riot -h dragonfly-cluster db-import "SELECT * FROM users" --url "jdbc:postgresql://x.x.x.x:5432/db" --username postgres --password postgres json.set --keyspace users --keys id`

Getting exception:
```
java.lang.IllegalStateException: No supported DataSource type found
        at org.springframework.util.Assert.state(Assert.java:76)
        at org.springframework.boot.jdbc.DataSourceBuilder$ReflectionDataSourceProperties.<init>(DataSourceBuilder.java:532)
        at org.springframework.boot.jdbc.DataSourceBuilder$DataSourceProperties.forType(DataSourceBuilder.java:334)
        at org.springframework.boot.jdbc.DataSourceBuilder.build(DataSourceBuilder.java:172)
        at com.redis.riot.db.DataSourceOptions.dataSource(DataSourceOptions.java:55)
        at com.redis.riot.db.DatabaseImport.reader(DatabaseImport.java:100)
        at com.redis.riot.db.DatabaseImport.job(DatabaseImport.java:94)
        at com.redis.riot.core.AbstractRiotCallable.call(AbstractRiotCallable.java:85)
        at com.redis.riot.cli.AbstractRiotCommand.call(AbstractRiotCommand.java:105)
        at com.redis.riot.cli.AbstractImportCommand.call(AbstractImportCommand.java:31)
        at com.redis.riot.cli.AbstractRiotCommand.call(AbstractRiotCommand.java:28)
        at picocli.CommandLine.executeUserObject(CommandLine.java:2041)
        at picocli.CommandLine.access$1500(CommandLine.java:148)
        at picocli.CommandLine$RunFirst.handle(CommandLine.java:2365)
        at picocli.CommandLine$RunFirst.handle(CommandLine.java:2329)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
        at picocli.CommandLine$RunFirst.execute(CommandLine.java:2331)
        at com.redis.riot.cli.AbstractMainCommand.executionStrategy(AbstractMainCommand.java:80)
        at com.redis.riot.cli.CompositeExecutionStrategy.execute(CompositeExecutionStrategy.java:28)
        at picocli.CommandLine.execute(CommandLine.java:2170)
        at com.redis.riot.cli.AbstractMainCommand.execute(AbstractMainCommand.java:66)
        at com.redis.riot.cli.AbstractMainCommand.run(AbstractMainCommand.java:42)
        at com.redis.riot.cli.Main.main(Main.java:9)
```

**Comment by smsajid** [comment]
This issue is also happening with the zip installation.
The issue is missing library HikariCP
Download and add to lib folder: https://repo1.maven.org/maven2/com/zaxxer/HikariCP/5.1.0/HikariCP-5.1.0.jar

**Comment by jruaux** [comment]
🎉 This issue has been resolved in `v4.0.0` ([Release Notes](https://github.com/redis/riot/releases/tag/v4.0.0))
